docs/release_checklist.md

```diff
@@ -86,6 +86,11 @@ If you do not have GPG already setup you can follow these steps:
 3. [Add key to GitHub account](https://help.github.com/en/articles/adding-a-new-gpg-key-to-your-github-account)
 4. [Set signing key in Git](https://help.github.com/en/articles/telling-git-about-your-signing-key)
 
+Once you have a signing key you need to add it to the KEYS file at the root of
```

Comment:
Why not just add the link to the KEYS file?

Reply:
The KEYS file is unavailable on the docs site. the docs should only link to itself.

Reply:
This is currently in the Helm repo but I'll replicate to the docs site once it goes in. This doc is in 2 places for the time being. I was trying to have just one copy that works for both locations. This may be the wrong assumption